app/src/main/java/org/sopt/dosopttemplate/presentation/home/HomeFragment.kt

```diff
@@ -0,0 +1,50 @@
+package org.sopt.dosopttemplate.presentation.home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.ConcatAdapter
+import org.sopt.dosopttemplate.databinding.FragmentHomeBinding
+import org.sopt.dosopttemplate.presentation.home.adapter.FriendAdapter
+import org.sopt.dosopttemplate.presentation.home.adapter.MyProfileAdapter
+import org.sopt.dosopttemplate.presentation.home.viewmodel.HomeViewModel
+
+class HomeFragment : Fragment() {
+    private var _binding: FragmentHomeBinding? = null
+    private val binding: FragmentHomeBinding
+        get() = requireNotNull(_binding) { "바인딩 객체가 생성되지 않음" }
+
+    private val viewModel: HomeViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View {
+        _binding = FragmentHomeBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val myProfileAdapter = MyProfileAdapter(requireContext()).apply {
+            setMyProfile(viewModel.profileData)
+        }
+
+        val friendAdapter = FriendAdapter(requireContext()).apply {
+            setFriendList(viewModel.mockFriendList)
+        }
+
+        val concatAdapter = ConcatAdapter(myProfileAdapter, friendAdapter)
+        binding.rvFriend.adapter = concatAdapter
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
```

Comment:
```suggestion
       _binding = null
        super.onDestroyView()
```
이렇게 처리하면  더 안정성있게 _binding 객체 메모리를 해제할 수 있다고 합니다.

Reply:
혹시 요 부분에 대한 아티클 같은 게 있을까요 ? 혹은 구글링 때 키워드 ... 힌트 부탁드ㄹㅕ요